automation/jenkins_build.sh

```diff
@@ -18,6 +18,46 @@ cleanup() {
 }
 trap 'cleanup fail' SIGINT SIGTERM
 
+deploy_build () {
+	local DEPLOY_DIR="$1"
+	local CHECK_COMPRESSED="$2"
+
+	local DEPLOY_ARTIFACT=$(jq --raw-output '.yocto.deployArtifact' $DEVICE_TYPE_JSON)
+	local COMPRESSED=$(jq --raw-output '.yocto.compressed' $DEVICE_TYPE_JSON)
+	local ARCHIVE=$(jq --raw-output '.yocto.archive' $DEVICE_TYPE_JSON)
+
+	rm -rf "$DEPLOY_DIR"
+	mkdir -p "$DEPLOY_DIR/image"
+
+	cp -v "$YOCTO_BUILD_DEPLOY/VERSION" "$DEPLOY_DIR"
+	cp -v "$YOCTO_BUILD_DEPLOY/VERSION_HOSTOS" "$DEPLOY_DIR"
+	cp -v "$DEVICE_TYPE_JSON" "$DEPLOY_DIR/device-type.json"
+
+	if [ $SLUG != "edge" ]; then
+		cp -v "$YOCTO_BUILD_DEPLOY/kernel_modules_headers.tar.gz" "$DEPLOY_DIR"
+
+		if [ "$CHECK_COMPRESSED" == "true" ]; then
+			if [ "${COMPRESSED}" == 'true' ]; then
+				if [ "${ARCHIVE}" == 'true' ]; then
```

Comment:
There is some strange combination of double quotes, single quotes and curly braces here. Unless you expect spaces you don't need quotes, so it should be as simple as:

    if [ $CHECK_COMPRESSED = "true" ]